docs/rules/padding-line-between-statements.md

```diff
@@ -0,0 +1,235 @@
+
+Examples of **incorrect** code for the `[{ blankline: "always", prev: ["const", "let", "var"], next: "*"}, { blankline: "any", prev: ["const", "let", "var"], next: ["const", "let", "var"]}]` configuration:
+
+```js
+/*eslint padding-line-between-statements: [
+    "error",
+    { blankline: "always", prev: ["const", "let", "var"], next: "*"},
+    { blankline: "any",    prev: ["const", "let", "var"], next: ["const", "let", "var"]}
+]*/
+
+function foo() {
+    var a = 0;
+    bar();
+}
+
+function foo() {
+    let a = 0;
+    bar();
+}
+
+function foo() {
+    const a = 0;
+    bar();
+}
+```
+
+Examples of **correct** code for the `[{ blankline: "always", prev: ["const", "let", "var"], next: "*"}, { blankline: "any", prev: ["const", "let", "var"], next: ["const", "let", "var"]}]` configuration:
+
+```js
+/*eslint padding-line-between-statements: [
+    "error",
+    { blankline: "always", prev: ["const", "let", "var"], next: "*"},
+    { blankline: "any",    prev: ["const", "let", "var"], next: ["const", "let", "var"]}
+]*/
+
+function foo() {
+    var a = 0;
+    var b = 0;
+
+    bar();
+}
+
+function foo() {
+    let a = 0;
+    const b = 0;
+
+    bar();
+}
+
+function foo() {
+    const a = 0;
+    const b = 0;
+
+    bar();
+}
+```
+
+----
+
+This configuration would require blank lines after all directive prologues, like the [lines-around-directive] rule.
+
+Examples of **incorrect** code for the `[{ blankline: "always", prev: "directive", next: "*" }, { blankline: "any", prev: "directive", next: "directive" }]` configuration:
+
+```js
+/*eslint padding-line-between-statements: [
+    "error",
+    { blankline: "always", prev: "directive", next: "*" },
+    { blankline: "any",    prev: "directive", next: "directive" }
+]*/
+
+"use strict";
+foo();
+```
+
+Examples of **correct** code for the `[{ blankline: "always", prev: "directive", next: "*" }, { blankline: "any", prev: "directive", next: "directive" }]` configuration:
+
+```js
+/*eslint padding-line-between-statements: [
+    "error",
+    { blankline: "always", prev: "directive", next: "*" },
+    { blankline: "any",    prev: "directive", next: "directive" }
+]*/
+
+"use strict";
+"use asm";
+
+foo();
+```
+
+## Compatibility
+
+- **JSCS:** [requirePaddingNewLineAfterVariableDeclaration]
+- **JSCS:** [requirePaddingNewLinesAfterBlocks]
+- **JSCS:** [disallowPaddingNewLinesAfterBlocks]
+- **JSCS:** [requirePaddingNewLinesAfterUseStrict]
+- **JSCS:** [disallowPaddingNewLinesAfterUseStrict]
+- **JSCS:** [requirePaddingNewLinesBeforeExport]
+- **JSCS:** [disallowPaddingNewLinesBeforeExport]
+- **JSCS:** [requirePaddingNewlinesBeforeKeywords]
+- **JSCS:** [disallowPaddingNewlinesBeforeKeywords]
+
+## When Not To Use It
+
+If you don't want to notify warnings about linebreaks, then it's safe to disable this rule.
+
+
+[lines-around-directive]: http://eslint.org/docs/rules/lines-around-directive
```

Comment:
So sorry! I totally misunderstood these. I didn't notice they were just the links 🙏